codesystems/CodeSystem-England-SDSJobRoleCode.xml

```diff
@@ -0,0 +1,27 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+    <id value="England-SDSJobRoleCode" />
```

Comment:
Retired Codesystem. To be changed in the valueset and examples.

Reply:
As this CodeSystem is retired. I will say it is safe to remove it from the project. Its not currently mapped to any element on PractitionerRole profile. The alternative for it in UK Core is ValueSet ['UKCorePracticeSettingCode'](https://simplifier.net/hl7fhirukcorer4/valueset-ukcore-practicesettingcode). Do update your examples with the alternative and ensure they included display so that we can see the specific role. 
E.g,
<system value="https://fhir.hl7.org.uk/CodeSystem/UKCore-PracticeSettingCode" />
            <code value="600" />
            <display value="General Medical Practice" />